src/types/tokens.ts

```diff
@@ -0,0 +1,25 @@
+import { 
+    TokenInstanceDetailDto,
+    TokenInstanceDto, 
+    TokenInstanceRequestDto, 
+    UpdateTokenInstanceRequest 
+} from "./openapi";
+import { 
+    AttributeRequestModel, 
+    AttributeResponseModel 
+} from "./attributes";
+
+export type TokenRequestDto = TokenInstanceRequestDto;
+export type TokenRequestModel = Omit<TokenRequestDto, "attributes | customAttributes"> & { attributes: Array<AttributeRequestModel>, customAttributes?: Array<AttributeRequestModel> };
+
+export type TokenUpdateRequestDto = TokenInstanceRequestDto;
+export type TokenUpdateRequestModel = Omit<TokenUpdateRequestDto, "attributes | customAttributes"> & { attributes: Array<AttributeRequestModel>, customAttributes?: Array<AttributeRequestModel> };
+
+export type TokenResponseDto = TokenInstanceDto;
+export type TokenResponseModel = TokenResponseDto;
+
+export type TokenDetailResponseDto = TokenInstanceDetailDto;
+export type TokenDetailResponseModel = Omit<TokenDetailResponseDto, "attributes | customAttributes"> & { attributes: Array<AttributeResponseModel>, customAttributes?: Array<AttributeResponseModel> };
```

Comment:
please add `status` and `metadata` and add them to transform function as well

Reply:
resolved